src/main/asciidoc/appendix/datatypes.adoc

```diff
@@ -29,5 +29,10 @@ It's a common <<_11-and-n1-embedded-relationships,one-to-one relationship>>|`RID
 |Byte|BYTE|Single byte.
 Useful to store small 8-bit signed integers|`java.lang.Byte` or `byte`|-128 -- +127|Any Number, String
 |Decimal|DECIMAL|Decimal numbers without rounding|`java.math.BigDecimal` | |Any Number, String
+| Point | POINT |2D shape type specified by two numbers.|  |  |
```

Comment:
There aren't new types yet, just functions. In the database now works by storing single values, like having 2 properties `lat` and `long` to store a point. I believe we need a proper type, but I still have to find the best solution out there.

Reply:
* Removed changes to type table in Section 11.1
* Appended descriptions in Section 8.1